### PR TITLE
housekeeping: Moving package signing into Cake script

### PR DESCRIPTION
**Pull request by RLittlesII**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Moves the code signing of ReactiveUI nuget packages into the Cake script

**What is the current behavior? (You can also link to an open issue here)**
Currently we call out to a powershell script from the Cake script.

**What is the new behavior (if this is a feature change)?**
The Cake script will call the global `SignClient` dotnet tool directly from Cake.

**What might this PR break?**
The code signing process.

**Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -319,9 +326,26 @@ Task("SignPackages")
     .WithCriteria(() => !isPullRequest)
     .Does(() =>
 {
-    StartPowershellFile("./SignPackages.ps1", args =>
+    var nupkgs = GetFiles(artifactDirectory + "/packages/*.nupkg");
```
Comment (glennawatson) [inline build.cake]: There is a constant for the packages directory above.

File: build.ps1 [hunk 2]
```diff
@@ -225,5 +225,6 @@ $cakeArguments += $ScriptArgs
 
 # Start Cake
 Write-Host "Running build script..."
+&$CAKE_EXE --bootstrap
```
Comment (glennawatson) [inline build.ps1]: What's bootstrap do?

Comment (RLittlesII) [reply build.ps1]: In the [Installation Section](https://www.gep13.co.uk/blog/introducing-cake.dotnettool.module#installation) it refers to needing to bootstrap cake for module inclusion.  It is a limitation that the Cake team is working to resolve so all you have to do is execute cake once in the future.

**Comment by RLittlesII** [comment]
@glennawatson I think this is read for final review.

**Pull request merged**